Pyrado/tests/test_environments.py

```diff
@@ -87,6 +89,8 @@
         "qq-su",
         "qcp-st",
         "qcp-su",
+        "qq-st-mj",
```

Comment:
you added the new env to the most relevant tests :+1:  please also add it to the test_setting_dp_vals() test, and locally check if it works